test/functional/sqlsrv/MsHelper.inc

```diff
@@ -359,6 +359,17 @@ function connect($options = array(), $disableCE = false)
         if (isColEncrypted()) {
             $connectionOptions = array_merge($connectionOptions, array("ColumnEncryption" => "Enabled"));
         }
+        if ($keystore == 'akv') {
+            $akv_options = array("KeyStoreAuthentication"=>$AKVKeyStoreAuthentication);
```

Comment:
Change keystore possible value for AKV from only `akv` to two diff choices (e.g. `akv_key`, `akv_pwd`, or something like this). Please make the corresponding changes in the Helper methods and/or constants.